data/json/emit.json

```diff
@@ -360,5 +360,14 @@
     "field": "fd_nuke_gas",
     "intensity": 3,
     "qty": 100
+  },
+  {
+    "id": "emit_sludge_puddle",
+    "type": "emit",
+    "//": "infrequent emission of sludge along the ground, made because sludge trail is too frequent",
+    "field": "fd_sludge",
+    "intensity": 3,
```

Comment:
This file still needs linting.
![image](https://user-images.githubusercontent.com/58050969/118740362-4f572c80-b811-11eb-9316-ed66b8a7ae80.png)

Reply:
ill fix that ASAP